code/modules/research/xenoarchaeology/finds/finds_misc.dm

```diff
@@ -25,7 +25,6 @@
 	return
 
 /obj/item/weapon/shard/plasma/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
-	..()
 	if ( istype(W, /obj/item/weapon/weldingtool))
```

Comment:
I think this might be worse than coders that leave spaces at both ends, both after and before parentheses.